components/formats-gpl/src/loci/formats/in/CellH5Reader.java

```diff
@@ -481,6 +495,20 @@ private void parseStructure() throws FormatException {
 
     for (int s=0; s<seriesNames.size(); s++) {
       store.setImageName(seriesNames.get(s), s);
+      
+      String plate_id =  MetadataTools.createLSID("Plate", 0);
+     
+      store.setPlateID(plate_id, 0);
+      store.setPlateName(seriesPlate.get(s), 0);
+      
+      String well_id =  MetadataTools.createLSID("Well", 0);
+      
+      store.setWellID(well_id, 0, 0);
+      store.setWellExternalIdentifier(seriesWell.get(s), 0, 0);
+      
+      String site_id = MetadataTools.createLSID("WellSample", 0);
+      store.setWellSampleID(site_id, 0, 0, 0);
+      store.setWellSampleIndex(NonNegativeInteger.valueOf(seriesSite.get(s)), 0, 0, 0);
```

Comment:
* We currently have the well information as a alpha numeric coordinate (e.g. B03). WellRow and WellColumn are stored as integers. I can implement a mapping (e.g. C12 will be 3x12; 1-based indexing?) Would this be okay?

* using store.setWellSampleImageRef() makes totally sense to me

thanks for all your help!